File: atcoder.jp/contests/abc168/tasks/abc168_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];
const print = (...messages: any) => console.log(...messages);

const take = (n: number, s: string) => s.slice(0, n);

const main = () => {
  const K = +input();
  const S = input();

  const ans = S.length <= K ? S : take(K, S) + "...";

  print(ans);
};

main();

File: atcoder.jp/contests/abc168/tasks/abc168_b/main.cpp
#include <bits/stdc++.h>
using namespace std;

string take(int n, const string &s) { return s.substr(0, n); }

int main() {
  int K;
  string S;
  cin >> K >> S;

  string ans = S.size() <= K ? S : take(K, S) + "...";

  cout << ans << endl;
}
